docs/CONTRIBUTING.md

```diff
@@ -9,6 +9,9 @@ Some suggestions to get started:
 - To use print debugging to the [Helix log file][log-file], you must:
   * Print using `log::info!`, `warn!`, or `error!`. (`log::info!("helix!")`)
   * Pass the appropriate verbosity level option for the desired log level. (`hx -v <file>` for info, more `v`s for higher severity inclusive)
+  * Want to display the logs in a separate file instead of using the `:log-open` command in your compiled Helix editor? Start your debug version with `cargo run -- --log foo.log` and in a new terminal use `tail -f foo.log`
+  * Instead of running a release version of Helix, while developping you may want to run in debug mode with `cargo run` which is way faster to compile
+  * Looking for even faster compile times? Give a try to [mold](https://github.com/rui314/mold)
```

Comment:
These are sub-bullets of the print debugging bullet but they should be top-level since they're unrelated

Reply:
You are right, thanks for spotting this mistake! I've corrected it now, and added another wording change. What do you think?